docs/ref/django-admin.txt

```diff
@@ -628,33 +640,31 @@ Example usage::
 
 .. django-admin-option:: --no-default-ignore
 
-Use the ``--no-default-ignore`` option to disable the default values of
-:djadminopt:`--ignore`.
+Disables the default values of ``--ignore``.
 
 .. django-admin-option:: --no-wrap
 
-Use the ``--no-wrap`` option to disable breaking long message lines into
-several lines in language files.
+Disables breaking long message lines into several lines in language files.
 
 .. django-admin-option:: --no-location
 
-Use the ``--no-location`` option to suppress writing '``#: filename:line``’
-comment lines in language files. Note that using this option makes it harder
-for technically skilled translators to understand each message's context.
+Suppresses writing '``#: filename:line``’ comment lines in language files.
+Using this option makes it harder for technically skilled translators to
+understand each message's context.
 
 .. django-admin-option:: --keep-pot
 
-Use the ``--keep-pot`` option to prevent Django from deleting the temporary
-``.pot`` files it generates before creating the .po file. This is useful for
-debugging errors which may prevent the final language files from being created.
+Prevents deleting the temporary ``.pot`` files generated before creating the
+``.po`` file. This is useful for debugging errors which may prevent the final
+language files from being created.
 
 .. seealso::
 
     See :ref:`customizing-makemessages` for instructions on how to customize
     the keywords that :djadmin:`makemessages` passes to ``xgettext``.
 
-makemigrations [<app_label>]
-----------------------------
+makemigrations [<app_label> ...]
+--------------------------------
```

Comment:
`loaddata` uses the `[fixture fixture ...]` form.

I personally prefer the `command [<arg> ...]` form better but as long as were consistent I don't mind either.